R/utilities.R

```diff
@@ -680,15 +688,13 @@ get_stan_model <- function() {
 #' @name rbmi-settings
 set_options <- function() {
 
-    cache_dir <- Sys.getenv("RBMI_CACHE_DIR")
-
-    if (cache_dir == "" || is.null(cache_dir)) {
-        cache_dir <- tools::R_user_dir("rbmi", which = "cache")
-    }
+    cache_dir <- Sys.getenv("RBMI_CACHE_DIR", unset = tools::R_user_dir("rbmi", which = "cache"))
+    enable_cache <- Sys.getenv("RBMI_ENABLE_CACHE", unset = "TRUE") == "TRUE"
```

Comment:
```suggestion
    enable_cache <- isTRUE(as.logical(Sys.getenv("RBMI_ENABLE_CACHE", unset = "TRUE")))
```
maybe use as.logical to catch `c("T", "TRUE", "True", "true")`